flarum/framework
# Do not catch exceptions when testing Console commands

**Changes proposed in this pull request:**

Configure the ConsoleApplication to not catch exceptions when testing Console commands.

This allows to easily test a case where a command should throw using `$this->expectException()`.

**QA**
For now I only tested it locally by modifying the vendor directory. It should not cause too much issues for existing testing suites, except if the way to check if a console was throwing was done by checking the output. As it is rather tedious I don't suspect that there are a lot of such test cases.

Edit: well there is one here :sweat_smile: 
https://github.com/flarum/framework/blob/af3f91ca5b30c2bfc7a7f9dd36839339372cc745/framework/core/tests/integration/extenders/ConsoleTest.php#L22-L29

This case has been simple to test as there is no other input, and Symfony seems to treat this exception differently, as usually it shows the file name and line of the throw statement. It can be written equally simply using `$this->expectExceptionMessage()`.

Edit2: In fact I think this test would even be better off using `$this->expectException()` as the goal is not to check the message but that the command is not found, and because this way the test is less brittle.

**Necessity**


**Confirmed**

- [ ] Backend changes: tests are green (run `composer test`).
